spec/features/resident_can_locate_problem_spec.rb

```diff
@@ -76,6 +76,11 @@
   end
 
   scenario 'changing the postcode after searching' do
+    fake_api = instance_double(JsonApi)
```

Comment:
If we don't care about the response from the API, could these three lines be replaced with the following..?

```
fake_api = instance_double(JsonApi, get: [])
```

Reply:
Could do, but with my Ben hat on, I feel like there's value in the 'symmetry' between the test cases: the philosophy is that if they're all built in the same way then it's easier to see what the differences are.

Reply:
👍